extensions/quest/src/actions/deleteQuest.tsx

```diff
@@ -0,0 +1,14 @@
+import { Action, Icon } from "@raycast/api";
+
+function DeleteQuestAction(props: { onDelete: () => void }) {
+  return (
+    <Action
+      icon={Icon.Trash}
+      title="Delete Quest"
```

Comment:
```suggestion
      title="Delete Quest"
      style={Action.Style.Destructive}
```
Could be nice to have a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) somewhere too 😊

Reply:
Added in d026d26df25cf65fb2ebc1bd3f1b4aed94c5aff8